### PR TITLE
added  JS file and onclick in HTML

### PR DESCRIPTION
**Pull request by Jomalagon**
@axelzo added JS file with function signin linked onclick at login in navbar.

### CODE REVIEW

File: scripFoodMenu.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    if(logged === true) {
+        for (let i = 0; i < newsFeed.length; i++) {
+            alert(newsFeed[i].username + "\n" + newsFeed[i].timeLine);
+        }
+        
+    } else {
+        alert("Your password or your user is wrong!");
+    }
+}
+
```
Comment (axelzo) [inline scripFoodMenu.js]: Avoid extra white space.

File: scripFoodMenu.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    // if (user === database[0].username &&
```
Comment (axelzo) [inline scripFoodMenu.js]: Avoid left commented code.

File: scripFoodMenu.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        username: "Juana",
+        timeLine: "so tired from all that learning"
+    },
+    {
+        username: "petra",
+        timeLine: "Javascript is sooo coool!"
+    }
+];
+
+
```
Comment (axelzo) [inline scripFoodMenu.js]: Avoid extra white space.

File: scripFoodMenu.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    }
+];
+
+var newsFeed = [
+    {
+        username: "Juana",
+        timeLine: "so tired from all that learning"
+    },
+    {
+        username: "petra",
```
Comment (axelzo) [inline scripFoodMenu.js]: Use Uppercase on `"petra"`.

File: index.html [hunk 5]
```diff
@@ -9,6 +9,7 @@
         <title>Food Menu | Good food</title>
     </head>
     <body>
+        <script type="text/javascript" src="scripFoodMenu.js"></script>
```
Comment (axelzo) [inline index.html]: You missed t on `scriptFoodMenu`

**Review by axelzo — CHANGES_REQUESTED** [review]

File: scripFoodMenu.js (hunk 1 above)
Comment (Jomalagon) [reply scripFoodMenu.js]: Ready

File: scripFoodMenu.js (hunk 3 above)
Comment (Jomalagon) [reply scripFoodMenu.js]: Ready

File: index.html (hunk 5 above)
Comment (Jomalagon) [reply index.html]: renamed to script.js

**Comment by Jomalagon** [comment]
@axelzo  I don't know what I did, but better than was if  stay.

**Pull request closed without merging**